static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,853 @@
+
+.hosted-page {
+    font-family: Helvetica, Arial sans-serif;
+}
+
+.hosted-page--bright.hosted-gallery-page .hosted-tone-btn,
+.hosted-page--bright .hosted__banner .hosted-tone-btn,
+.hosted-page--bright .hosted-tone-btn:focus {
+    color: #ffffff;
+}
+
+.hosted-page--bright.hosted-gallery-page .hosted-tone-btn svg,
+.hosted-page--bright .hosted__banner .hosted-tone-btn svg,
+.hosted-page--bright .hosted-tone-btn:focus svg {
+    fill: #ffffff;
+}
+
+.hosted-page--bright .hostedbadge.hosted-tone-bg .hostedbadge__info {
+    color: #ffffff;
+}
+
+.hosted-page--bright .survey-text__header {
+    color: #ffffff;
+}
+
+.hosted-page--bright .survey-text__header .site-message__close-btn {
+    border-color: #ffffff;
+}
+
+.hosted-page--bright .survey-text__header .inline-cross {
+    fill: #ffffff;
+}
+
+.hosted__glogo {
+    display: inline-block;
+    padding: 4px 10px;
+    background-color: #005689;
+    text-align: right;
+}
+
+.hosted__glogo svg {
+    fill: #ffffff;
+    width: 115px;
+    height: 40px;
+}
+
+.hosted__glogo .inline-logo {
+    display: inline-block;
+    margin-top: -5px;
+}
+
+.hosted__glogotext {
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: .5px;
+    line-height: 16px;
+    margin: 0;
+    text-align: left;
+    color: rgba(255, 255, 255, .5);
+}
+
+.hostedbadge {
+    position: absolute;
+}
+
+.hostedbadge__logo.hostedbadge__logo {
+    position: relative;
+}
+
+.hosted__heading {
+    margin: 0;
+    font-size: 22px;
+    line-height: 24px;
+}
+
+.hosted__banner {
+    height: 200px;
+    position: relative;
+    background-size: cover;
+    background-position: left;
+}
+
+.hosted__banner:after {
```

Comment:
Would we be able to use `&:after` with the block above? I find scss much easier to read. Also much easier to create DRY code.